src/Mcamara/LaravelLocalization/LaravelLocalization.php

```diff
@@ -47,7 +47,7 @@ class LaravelLocalization
     /**
      * Current locale.
      *
-     * @var string
+     * @var string|false
      */
     protected $currentLocale = false;
```

Comment:
How about remove comment and just add
```suggestion
    protected string|bool $currentLocale = false;
```
?

Reply:
I see you did that on other branch, nvm